Support react 18

```
▶ npm i -SE use-debouncy  
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: client@1.0.0
npm ERR! Found: react@18.2.0
npm ERR! node_modules/react
npm ERR!   react@"18.2.0" from client@1.0.0
npm ERR!   packages/client
npm ERR!     client@1.0.0
npm ERR!     node_modules/client
npm ERR!       workspace packages/client from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.8.0 || ^17.0.0" from use-debouncy@4.3.0
npm ERR! node_modules/use-debouncy
npm ERR!   use-debouncy@"*" from client@1.0.0
npm ERR!   packages/client
npm ERR!     client@1.0.0
npm ERR!     node_modules/client
npm ERR!       workspace packages/client from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/marcusstenbeck/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/marcusstenbeck/.npm/_logs/2022-09-08T20_39_11_326Z-debug-0.log
```

--- a/.npmrc
+++ b/.npmrc
@@ -1,3 +1,4 @@
 @bit:registry=https://node.bit.dev
 always-auth=true
 save-exact=true
+legacy-peer-deps=true

